### DISCUSSION TITLE
Ribbon Bar

### DISCUSSION THREAD

**Discussion by mastercodefu** [vanjarosoftware/Vanjaro.Platform#1835] [Q&A]
Is there a way to switch back to the default ribbon bar after have completed theme?

**Comment by meetmandeep** [comment]
@mastercodefu I'm not sure if I understand the question. Are you using Vanjaro for DNN? If so, if you navigate to a page that is not created by Vanjaro then the default DNN Persona Bar will appear. Otherwise, the Persona Bar is replaced with Vanjaro Menu.

**Comment by mastercodefu** [comment]
Hi Mandeep, first great work btw on the project i'm using dnn and as mentioned in the past have felt this is missing from dnn skinning engine. The ability to create or design skins directly in dnn similiar to wordpress.  Your method is so much more affective and would be a great addition to working with skins/themes etc but then still having the default framework beind also.   If you get my meaning.

Craig
